extensions/asana/src/components/CreateTaskForm.tsx

```diff
@@ -126,12 +132,24 @@ export default function CreateTaskForm(props: {
       }
       enableDrafts={!props.fromEmptyView}
     >
-      <Form.Dropdown title="Workspace" storeValue {...itemProps.workspace}>
+      <Form.Dropdown title="Workspace" storeValue  {...itemProps.workspace}>
         {workspaces?.map((workspace) => {
           return <Form.Dropdown.Item key={workspace.gid} value={workspace.gid} title={workspace.name} />;
         })}
       </Form.Dropdown>
 
+      <Form.Dropdown title="Teams" storeValue autoFocus {...itemProps.team}>
```

Comment:
If `storeValue` is used for teams, then the `autofocus` should be kept for the "Task Name" field as the team value will be remembered for the subsequent creations.

Reply:
I tried putting autoFocus on the Task Name like you said but it still wouldn't remember the value for Teams or Projects for the next time I create a task.

Reply:
And it does remember the value when there's `autoFocus` on the projects field? It doesn't seem related to me 🤔